src/librustc_session/options.rs

```diff
@@ -142,6 +142,9 @@ top_level_options!(
         // by the compiler.
         json_artifact_notifications: bool [TRACKED],
 
+        // `true` if we're emitting a JSON blob containing the unused externs
+        json_unused_externs: bool [TRACKED],
```

Comment:
Does this need to be tracked?

Reply:
I wanted to be on the safe side but i guess it can be UNTRACKED.